.github/dependabot.yaml

```diff
@@ -0,0 +1,16 @@
+version: 2
+
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    labels: ["area/ci", "dependencies"]
+    schedule:
+      # By default, this will be on a monday.
+      interval: "weekly"
+    groups:
```

Comment:
another useful option might be using [reviewers](https://github.com/weaveworks/pipeline-controller/pull/190/files#diff-dd4fbda47e51f1e35defb9275a9cd9c212ecde0b870cba89ddaaae65c5f3cd28R7) so PRs are assigned for reviewing reducing the risk or staying there without review for not being assigned? i guess that is up you to decide what works best